base_restrict_user_fields/models/res_users.py

```diff
@@ -0,0 +1,15 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+
+from odoo import fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    restrict_fields = fields.Boolean(string="Restrict fields", compute="_compute_restrict_fields", default=False)
```

Comment:
Rename according actual meaning, and default attribute makes no sense, because field is fully computed (non stored):
```suggestion
    unrestrict_fields = fields.Boolean(string="Don't restrict fields", compute="_compute_unrestrict_fields")
```

Reply:
Still pending removing `default=False`.